### PR TITLE
add js library

### PR DESCRIPTION
**Pull request by philschmid**

### CODE REVIEW

File: docs/source/guides/test_endpoint.mdx [hunk 1]
```diff
@@ -1,11 +1,22 @@
 # Send Requests to Endpoints
 
-The Endpoint overview provides access to the Inference Widget which can be used to send requests (see step 6 of [Create an Endpoint](/docs/inference-endpoints/guides/create_endpoint)). This allows you to quickly test your Endpoint with different inputs and share it with team members. In addition, the Inference Widget has an additional option for creating your request as a cURL command and copying it so you can easily share or use it.
+You can send requests to Inference Endpoints using the UI leveraging the Inference Widget or programmatically, e.g. with cURL or the `@huggingface/inference` library. Below you can find examples for each of these methods.
 
-<img src="https://raw.githubusercontent.com/huggingface/hf-endpoints-documentation/main/assets/2_copy_curl.png" alt="copy curl" />
+## Use the UI to send request
```
Comment (Michellehbn) [inline docs/source/guides/test_endpoint.mdx:5]:
```suggestion
## Use the UI to send requests
```

File: docs/source/guides/test_endpoint.mdx [hunk 2]
```diff
@@ -27,3 +38,30 @@ curl --request POST \
 ```
 
 To use your cURL command as code, use the [cURL Converter](https://curlconverter.com/) tool to quickly get started with the programming language of your choice.
+
+## Use javascript library `@huggingface/inference`
```
Comment (Michellehbn) [inline docs/source/guides/test_endpoint.mdx:42]:
```suggestion
## Use `@huggingface/inference`
```

File: docs/source/guides/test_endpoint.mdx [hunk 3]
```diff
@@ -27,3 +38,30 @@ curl --request POST \
...
+
+here's how you can call an inference endpoint:
```
Comment (Michellehbn) [inline docs/source/guides/test_endpoint.mdx:44]:
```suggestion
You can use the javascript library to call an inference endpoint:
```

File: docs/source/guides/test_endpoint.mdx (hunk 2 above)
Comment (Michellehbn) [reply docs/source/guides/test_endpoint.mdx:42]: or maybe "Use the javascript library"

**Review by Michellehbn — COMMENTED** [review]
looking good!!!

File: docs/source/guides/test_endpoint.mdx [hunk 4]
```diff
@@ -27,3 +38,30 @@ curl --request POST \
...
+
+const gpt2 = inference.endpoint('https://xyz.eu-west-1.aws.endpoints.huggingface.cloud/gpt2-endpoint')
+const { generated_text } = await gpt2.textGeneration({ inputs: 'The answer to the universe is' })
+```
+
+**custom handler**
+
+`@huggingface/inference` supports tasks from https://huggingface.co/tasks, and is typed accordingly. But what if your model as additional inputs, or even custom inputs or outputs?
+
+You can use `.request` and `.streamingRequest` now!
```
Comment (coyotte508) [inline docs/source/guides/test_endpoint.mdx:57]:
```suggestion
### Custom handler

`@huggingface/inference` supports tasks from https://huggingface.co/tasks, and is typed accordingly. 

If your model has additional inputs, or even custom inputs / outputs you can use the more generic `.request` / `streamingRequest`:
```

The release notes are in a "peppy" tone, the doc should be more relaxed / neutral :)

**Pull request merged**